msu/hooks/skills/skill_container.nut

```diff
@@ -545,12 +544,4 @@
 
 		return ret;
 	}
-
-	foreach (event in ::MSU.Skills.EventsToAdd)
-	{
-		local name = event.Name;
-		local update = event.Update;
-		local aliveOnly = event.AliveOnly;
-		o[name] <- @(...) this.callSkillsFunction(name, vargv, update, aliveOnly);
-	}
```

Comment:
Not sure where this got moved

Reply:
This got moved to `::MSU.Skills.addEvent` where we now add via `compileString`.